src/app/tests/suites/TV_MediaPlaybackCluster.yaml

```diff
@@ -0,0 +1,65 @@
+# Copyright (c) 2021 Project CHIP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Media Playback Cluster Tests
+
+config:
+    cluster: "Media Playback"
+    endpoint: 3
+
+tests:
+    - label: "Media Playback Play Command"
```

Comment:
Should the commands here assert something about the status sent back in the response?  Followup is fine.

Reply:
Added assertion of success response.